runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.module.scss

```diff
@@ -0,0 +1,21 @@
+@import "#src-landing/scss/main";
+
+.video{
+    display: flex;
+
+    justify-content: center;
+
+    margin-block: 10px;
+
+    width: 100%;
+    height: 100%;
+}
+
+.embed{
```

Comment:
![image](https://github.com/runbotics/runbotics/assets/69820069/272afee5-5621-4d68-bfae-1a172f8ff232)
space is beautiful

remember that